SpatialOS/scripts/convert_snapshot.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# This script converts a binary snapshot file to its text format representation
+# Usage: convert_snapshot.sh <binary snapshot file name, optional, default value: "default.snapshot"> <text snapshot file name, optional, default value: "default.txt">
```

Comment:
The `optional`s look confusing to me - if I understand the script correctly, you can't pass `text snapshot file name` without passing `binary snapshot file name`, so in a way, it's not optional? 🤔

Reply:
Discussed offline.